src/main/kotlin/io/emeraldpay/dshackle/rpc/NativeSubscribe.kt

```diff
@@ -97,7 +97,11 @@ open class NativeSubscribe(
     }
 
     open fun subscribe(chain: Chain, method: String, params: Any?, matcher: Selector.Matcher): Flux<out Any> =
-        getUpstream(chain).getEgressSubscription().subscribe(method, params, matcher)
+        getUpstream(chain).getEgressSubscription()
+            .subscribe(method, params, matcher)
+            .doOnError {
+                log.error("Error during subscription", it)
```

Comment:
maybe add some context like chain and method?

Reply:
Done